core/src/main/scala/org/apache/spark/executor/OutputMetrics.scala

```diff
@@ -20,7 +20,6 @@ package org.apache.spark.executor
 import org.apache.spark.annotation.DeveloperApi
 import org.apache.spark.util.LongAccumulator
 
-
```

Comment:
why are you removing the line here? having two blank lines to separate top level classes is actually a style.